src/System.Private.CoreLib/src/System/RtType.cs

```diff
@@ -3274,7 +3200,7 @@ public override bool IsAssignableFrom(Type c)
             }
 
             // Special case for TypeBuilder to be backward-compatible.
-            if (c is System.Reflection.Emit.TypeBuilder)
```

Comment:
It would be better to keep the explicit `System.Reflection.Emit` references here instead of `using` at the top.

Reply:
Really? I've not seen much precedent for this anywhere in corefx or coreclr where there is no ambiguity. It's fairly obvious to me and I assume most people who know anything about reflection that `TypeBuilder` is in an `Emit` case